va/va.h

```diff
@@ -3812,6 +3812,36 @@ VAStatus vaMapBuffer(
     void **pbuf     /* out */
 );
 
+/**
+ * /brief Map data store of the buffer into the client's address space
+ *
+ * vaMapBuffer2 is similar with vaMapBuffer, but with addtional parameters "flags"
+ * the flags is defined as VA_MAP_XXX. Applications may combine one or more of these flags.
+ * if the flags equal to 0, the behavior should be same as vaMapBuffer.
+ *
+ * VA_MAP_READ / VA_MAP_WRITE is defined to identifies a resource to be accessed 
+ * for reading and writing by the CPU.
+ * if only set VA_MAP_READ without VA_AMP_WRITE, backend driver could not guarantee
```

Comment:
1. Capital `I`
2. Type in VA_MAP_WRITE
3. "may not" instead of "could not"

Should we be more directive something like:
```
If only set VA_MAP_READ without VA_MAP_WRITE, user must assure that there will be no writing to the resource. Backend driver may not guarantee correctness of data writing.
```
And same vice versa?